composer.json

```
@@ -37,9 +37,11 @@
         "webmozart/assert": "^1.11"
     },
     "require-dev": {
+        "ext-pdo": "*",
         "codewithkyrian/chromadb-php": "^0.2.1 || ^0.3 || ^0.4",
         "codewithkyrian/transformers": "^0.5.3",
         "async-aws/bedrock-runtime": "^0.1.0",
+        "doctrine/dbal": "^4.2",
```

could also be, right?
```suggestion
        "doctrine/dbal": "^3.0 || ^4.0",
```

we won't see a difference tho without example or test :D